flang/test/Driver/linker-flags.f90

```diff
@@ -50,3 +50,19 @@
 ! MSVC-SAME: FortranDecimal.lib
 ! MSVC-SAME: /subsystem:console
 ! MSVC-SAME: "[[object_file]]"
+
+! Verify that certain linker flags are known to the frontend and are passed on
+! to the linker.
+
+! RUN: %flang -### -rpath /path/to/dir %s 2>&1 \
```

Comment:
Consider using one command to test many options that can be used together.

Note that you can also use -static -shared together: 408e6de8c09fc7e71329199947e977fe1c40caf1 , so I'm fine if you use one line for -shared and another for -static :)

Reply:
The tests are trivial but having too many tests slows down execution. It certainly doesn't matter in this case but if everybody adds more tests than necessary, the testsuite execution will be very slow. In this case, just test all options in one command.

Reply:
Fair enough. Done.